google-cloud-tools-plugin/build.gradle

```diff
@@ -37,14 +37,19 @@ dependencies {
     compile(project(':google-account-plugin'))
     compile(project(':common-lib'))
 
-    compile files('lib/google-api-services-clouddebugger-v2-rev4-1.20.0.jar')
-    compile files('lib/google-api-services-cloudresourcemanager-v1beta1-rev12-1.21.0.jar')
+    compile 'com.google.apis:google-api-services-clouddebugger:v2-rev7-1.21.0'
+    compile 'com.google.apis:google-api-services-cloudresourcemanager:v1beta1-rev12-1.21.0'
     compile files('lib/google-api-services-source.jar')
 
     testCompile(project(':common-test-lib'))
     testRuntime files('../google-account-plugin/lib/google-gct-login-context-ij-pg.jar')
 }
 
+configurations {
```

Comment:
does it make sense to add this to our provided checks?

https://github.com/GoogleCloudPlatform/gcloud-intellij/blob/master/build.gradle#L65